webapp/src/components/Editor/DrawingBoard/Canvas/Worker/SelectorWorker.ts

```diff
@@ -101,6 +102,15 @@ class SelectorWorker extends Worker {
     });
     return target;
   }
+
+  /**
+   * moveToFront move selected shape at the head
+   */
+  moveToFront(shape: Shape): void {
+    this.update((root: Root) => {
+      root.shapes.moveLast(shape.getID());
+    });
+  }
```

Comment:
There seems to be a bug related to this feature.

```
2021-10-29T23:55:52.695Z	FATAL	fail to find the given createdAt: 9:1:617c89d76b2c40c89144e34b
github.com/yorkie-team/yorkie/pkg/document/json.(*RGATreeList).findNextBeforeExecutedAt
	/app/pkg/document/json/rga_tree_list.go:312
github.com/yorkie-team/yorkie/pkg/document/json.(*RGATreeList).insertAfter
	/app/pkg/document/json/rga_tree_list.go:350
github.com/yorkie-team/yorkie/pkg/document/json.(*RGATreeList).MoveAfter
	/app/pkg/document/json/rga_tree_list.go:267
github.com/yorkie-team/yorkie/pkg/document/json.(*Array).MoveAfter
	/app/pkg/document/json/array.go:70
github.com/yorkie-team/yorkie/pkg/document/operation.(*Move).Execute
	/app/pkg/document/operation/move.go:63
github.com/yorkie-team/yorkie/pkg/document/change.(*Change).Execute
	/app/pkg/document/change/change.go:48
github.com/yorkie-team/yorkie/pkg/document.(*InternalDocument).applyChanges
	/app/pkg/document/internal_document.go:208
github.com/yorkie-team/yorkie/pkg/document.(*InternalDocument).ApplyChangePack
	/app/pkg/document/internal_document.go:110
github.com/yorkie-team/yorkie/yorkie/packs.storeSnapshot
	/app/yorkie/packs/packs.go:408
github.com/yorkie-team/yorkie/yorkie/packs.PushPull.func1
	/app/yorkie/packs/packs.go:150
github.com/yorkie-team/yorkie/yorkie/backend.(*Backend).AttachGoroutine.func1
	/app/yorkie/backend/backend.go:164
```

How about proceeding with this PR as below:
1. Remove the shape order feature from this PR.
2. Create an issue for the bug.

Reply:
@hackerwins  I processed suggestion 1